utils/handleVote.js

```diff
@@ -42,16 +42,16 @@ export default function handleVote(currentArticle, voteDirection, hasVoted, setH
         setHasVoted("")
         voteOnArticle(currentArticle.article_id, 1).catch((err) => {
             setIsError(true)
-            setCurrentVotes(currentVotes + 1)
+            setCurrentVotes(currentVotes - 1)
         })
     }
 
     if (voteDirection === "down" && hasVoted === "up") {
```

Comment:
nicely handled logic of being able to use the up/down vote options in tandem with each other!